vendor/github.com/riobard/go-virtualbox/machine.go

```diff
@@ -38,6 +38,10 @@ const (
 	F_vtxvpid
 	F_vtxux
 	F_accelerate3d
+
+	F_usb
```

Comment:
__[golint]__
* [golint] don't use underscores in Go names; const F_usb should be FUsb

<!-- Comment by [SideCI](https://sideci.com) -->